charts/keycloakx/examples/postgresql-ingress/keycloak-server-values.yaml

```diff
@@ -0,0 +1,78 @@
+# This is an example configuration, for production grade configuration see the Keycloak documentation.
+# See https://www.keycloak.org/server/configuration
+# See https://www.keycloak.org/server/all-config
+command:
+  - "/opt/keycloak/bin/kc.sh"
+  - "--verbose"
+  - "start"
+  - "--auto-build"
+  - "--http-enabled=true"
+  - "--http-port=8080"
+  - "--hostname-strict=false"
+  - "--hostname-strict-https=false"
+  - "--spi-events-listener-jboss-logging-success-level=info"
+  - "--spi-events-listener-jboss-logging-error-level=warn"
+  - "--proxy=edge"
+
+# Configure ingress
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
Shouldn't we use the ingressClassName field instead of the "deprecated" annotation here?
see https://github.com/codecentric/helm-charts/blob/master/charts/keycloakx/templates/ingress.yaml#L21